core/src/test/java/ai/timefold/solver/core/api/solver/SolutionManagerTest.java

```diff
@@ -338,7 +338,7 @@ void analyzeWithUninitializedSolution(SolutionManagerSource SolutionManagerSourc
 
     @ParameterizedTest
     @EnumSource(SolutionManagerSource.class)
-    void recommendFit(SolutionManagerSource SolutionManagerSource) {
+    void recommendAssignment(SolutionManagerSource SolutionManagerSource) {
```

Comment:
Maybe we should create a separate test for that. Even though `recommendFit` is deprecated now, we still should test it to ensure no regressions are introduced.